test/benchmark/generation/TransitivityMatrixGraph.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.core.test.benchmark.generation;
+
+import com.vaticle.typedb.core.TypeDB;
+import com.vaticle.typedb.core.common.parameters.Arguments;
+import com.vaticle.typedb.core.common.parameters.Label;
+import com.vaticle.typedb.core.concept.thing.Relation;
+import com.vaticle.typedb.core.concept.thing.Thing;
+import com.vaticle.typedb.core.concept.type.RelationType;
+import com.vaticle.typedb.core.concept.type.RoleType;
+import com.vaticle.typedb.core.reasoner.benchmark.Util;
+
+@SuppressWarnings("CheckReturnValue")
+public abstract class TransitivityMatrixGraph {
+
+    private final TypeDB.DatabaseManager dbm;
```

Comment:
our name for this variable is almost always `databaseMgr`

Reply:
it would also be a good idea to close the manager/databases after the test is complete?

Reply:
That's the responsibility of whoever creates it.
Which are the tests which use these classes.